product_docs/docs/pgd/5/routing/index.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "Application connection management"
+navTitle: "Connection management"
+indexCards: none
+---
+
+
+Managing application connections is important part of high availability.
+
+Especially with asynchronous replication, having consistent write lead node is
+important in order to avoid conflicts and guarantee availability for the
+application.
+
+EDB Postgres Distributed provides a proxy layer called PDG-Proxy which is
+normally installed in highly available configuration (at least 2 instances per
+region).
+
+The PGD-Proxy connects to one of the EDB Postgres Distributed nodes and monitors
+routing configuration changes as decided by the EDB Postgres Distributed cluster
+and ensures that the connections are routed to correct node(s) consistently.
+
+## Configuration
+
+The configuration of the routing is done through either SQL interfaces or through
+PGD-CLI.
+
+The SQL interfaces are described in this section.
+
+You can enable routing decisions by calling `bdr.alter_node_group_option()` function.
+For example `SELECT bdr.alter_node_group_option('region1-group', 'enable_proxy_routing', 'true')`.
+It can be disabled again by setting the same option to `false`.
+
+There are additional group-level options that affect the routing decisions:
+
+- route_writer_max_lag - Maximum lag in bytes of the new write candidate to be
+  selected as write leader, if no candidate passes this, there will be no writer
+  selected automatically.
+- route_reader_max_lag - Maximum lag in bytes for node to be considered viable
+  read-only node (currently reserved for future use).
+- route_writer_wait_flush - Whether to wait for replication queue flush before
```

Comment:
I think as per the discussion in [BDR-2752](https://enterprisedb.atlassian.net/browse/BDR-2752) route_writer_wait_flush is not used anymore and can be removed from here.